<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popUpDialogClass</class>
 <widget class="QWidget" name="popUpDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="FloatingText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Got Time?</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="FloatingBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>256</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Shortcuts">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>90</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ShortcutsBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>108</y>
     <width>256</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <zorder>FloatingText</zorder>
  <zorder>Shortcuts</zorder>
  <zorder>FloatingBox</zorder>
  <zorder>ShortcutsBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
